src/main/java/com/checkmarx/flow/dto/report/AnalyticsReport.java

```diff
@@ -1,29 +1,29 @@
 package com.checkmarx.flow.dto.report;
 
 import com.checkmarx.flow.dto.ScanRequest;
-import groovy.util.logging.Slf4j;
+import com.checkmarx.flow.utils.AesEncodingUtils;
+import com.checkmarx.sdk.exception.CheckmarxException;
 import lombok.Data;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 import static net.logstash.logback.marker.Markers.*;
 
 @Data
-@Slf4j
 public abstract class AnalyticsReport {
 
-    public static final Logger jsonlogger = LoggerFactory.getLogger("jsonLogger");
+    protected static final Logger jsonlogger = LoggerFactory.getLogger("jsonLogger");
+    protected static final Logger log = org.slf4j.LoggerFactory.getLogger(AnalyticsReport.class);
     
     public static final String SAST = "SAST";
     public static final String OSA = "OSA";
 
+    protected String repoUrl = null;
+    protected String scanInitiator;
     protected String scanId;
-    protected String repoUrl;
-    protected String scanType;
     
     public AnalyticsReport(String scanId, ScanRequest request) {
         this.scanId = scanId;
-        this.repoUrl = request.getRepoUrl();
-        scanType = OSA;
+        scanInitiator = OSA;
     }
```

Comment:
In the SAST constructor you check the scan Id for null value, in here, what if the String scan Id is empty?